server/docker-base/template.build.gradle

```diff
@@ -24,6 +24,8 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    namespace = "com.defold.defold"
```

Comment:
New requirement in latest gradle android plugin. https://developer.android.com/build/configure-app-module#set-namespace